tests/test_basic_ops_dict.py

```diff
@@ -255,10 +255,13 @@ def foo(a: int, b: int) -> int:
     assert 12 == foo(**c)
 
 
-def test_items() -> None:
+def test_items_iterator_behavior() -> None:
     c = OmegaConf.create({"a": 2, "b": 10})
-    assert sorted([("a", 2), ("b", 10)]) == sorted(list(c.items()))
+    assert list(c.items()) == [("a", 2), ("b", 10)]
 
+    # This is actually not compatible with native dict:
+    # Due to implementation considerations, DictConfig items() returns a list.
+    # If that can be fixed, feel free to remove this block
```

Comment:
There's no reason why it couldn't be changed, is there?

Reply:
What couldn't be changed?

Reply:
If I understand you correctly:
The point of this comment is to say you fixing items to be more consistent is fine. this would require a change to this test.

Reply:
Ok, the way I read it ("If ithat can be fixed"), it sounded like something that maybe would be difficult to change, and I couldn't see why. Anyway, not a big deal => resolving.